src/utils/globals.ts

```diff
@@ -28,7 +35,10 @@ export const inform = Debug("ghas:inform") as Debug.Debugger;
 export const error = Debug("ghas:error") as Debug.Debugger;
 export const destDir = "Desktop" as string;
 export const windestDir = "Documents" as string;
-export const user = process.cwd().split("/")[2] as string;
+export const user =
+    isLinux === true
+      ? process.env.USER
+      : process.cwd().split("/")[2] as string;
 export const winUser = process.cwd().split("\\")[2] as string;
```

Comment:
isn't this line now redundant now? 🤔 Here you are encapsulating that logic:

```javascript
export const user =
    isLinux === true
      ? process.env.USER
      : process.cwd().split("/")[2] as string;
```

I think we can go ahead and remove this line now, right, and then update:

```
      cwd: `/Users/${winUser}/${windestDir}/${tempDIR}/${repo}`,
```

the just be `User`? 🤔

Reply:
You are right that this should probably be changed in some way 😄 💯 I saw that one windows command was set to `${user}` while all others were set to `${winUser}` so I changed that one but I didn't make any other changes. Not sure if we can remove winUser, because in user we split on "/" (in case of Mac I guess) and in winUser on "\". I can just add a check to for windows to `user`.